plugins/ipam/commands/commands.go

```diff
@@ -28,6 +28,7 @@ import (
 // Add will return ip, gateway, routes which can be
 // used in bridge plugin to configure veth pair and bridge
 func Add(args *skel.CmdArgs) error {
+	defer seelog.Flush()
```

Comment:
Can we switch the alias from `seelog` to `log` and make it consistent across all our packages ?